README.rdoc

```diff
@@ -96,6 +96,7 @@ needs. To make your life easier, TeX source and log are attached to the exceptio
 
 * ruby 1.8 or 1.9
 * rails 3 or later
+* LaTex
```

Comment:
```suggestion
* TeX, LaTeX or XeTeX depending on your settings (the default is using `pdflatex`)
```

Reply:
Oh, I totally missed your response in the flood of my work GH notifications. I'm just going through my open PRs and committed the suggestion. Thanks!